scala/scala.bzl

```diff
@@ -147,7 +137,7 @@ rm -rf {out}_tmp
     outs.extend([ctx.outputs.ijar])
   ctx.action(
       inputs=list(jars) +
-          list(dep_srcjars) +
+          list(all_srcjars) +
           ctx.files.srcs +
```

Comment:
it should be all 3. ctx.files.srcs goes to sources and srcjars, and dep_srcjars came from ctx.attr.deps